textarea/elfinder-sakai/src/webapp/WEB-INF/applicationContext.xml

```diff
@@ -27,13 +27,13 @@
                             <property name="contentHostingService" ref="org.sakaiproject.content.api.ContentHostingService"/>
                             <property name="siteService" ref="org.sakaiproject.site.api.SiteService"/>
 						</bean>
-						<!--
-						  This is disabled because it isn't doing permission checks but it's been useful in development
-						  to have multiple tools working.-->
 						<bean class="org.sakaiproject.elfinder.sakai.msgcntr.MsgCntrSiteVolumeFactory">
 							<property name="discussionForumManager" ref="org.sakaiproject.api.app.messageforums.ui.DiscussionForumManager"/>
 							<property name="messageForumsForumManager" ref="org.sakaiproject.api.app.messageforums.MessageForumsForumManager"/>
 							<property name="uiPermissionsManager" ref="org.sakaiproject.api.app.messageforums.ui.UIPermissionsManager"/>
+						</bean>
```

Comment:
I think you've removed the userDirectoryService from the MsgCntrSiteVolumeFactory.